api/src/helpers.py

```diff
@@ -58,39 +75,50 @@ def get_status(dag_run_id: str) -> Dict:
         f"https://{mwaa_cli_token['WebServerHostname']}/aws_mwaa/cli"
     )
 
-    raw_data = "dags list-runs -d veda_discover"
-    mwaa_response = requests.post(
+    raw_data_ingest = "dags list-runs -d veda_ingest"
```

Comment:
Did we decide to not use the REST api? https://airflow.apache.org/docs/apache-airflow/stable/stable-rest-api-ref.html#operation/get_dag_run

Reply:
Nope, I think the REST API is a great idea here... I wrote this before I knew of this feature 🤦

Reply:
I can stamp this if you want to punt on that. Also happy to help with that.

Reply:
After a bit of digging, it looks like the API is not something AWS gives people access to and that the CLI-backed API is what we've got (see comments here: https://stackoverflow.com/questions/67884770/is-it-possible-to-access-the-airflow-api-in-aws-mwaa). It doesn't lend itself to the clearest code, but it appears to be sufficient to our needs